index.html

```diff
@@ -679,19 +678,19 @@ <h4>NewPipe is free and open source</h4>
                     <h3 class="title">Support NewPipe</h3>
                 </div>
                 <div class="col-md-6 tile tile-left" id="tile-free">
-                    <h4>Why NewPipe is free</h4>
+                    <h4>Why NewPipe is libre and gratis</h4>
```

Comment:
Aren't they opposites? https://en.wikipedia.org/wiki/Gratis_versus_libre

Reply:
Hence the _and_. Adding Oxford comma to drive the point home.

Reply:
I don't think it is needed.

Reply:
@opusforlife2 The idea is to get donations, by way of making people understand why some software is different.
It is consistent, and it works.